### DISCUSSION TITLE
Figspec v2

### DISCUSSION THREAD

**Discussion by pocka** [storybookjs/addon-designs#212] [Show and tell]
Just to let you know, I published Figspec v2 ([demo](https://figspec.netlify.app/)), which improves performance and UI, also adds user preference system.
Changes affecting to this addon are:

- `nodes` property and `documentNode` property are renamed to `apiResponse`.
- Addition of preferences ("What CSS length unit/color notation is used for code generation?" "Enable color preview inside CSS code?" etc)
  + Need to persist the preference to browser storage otherwise it'll be reset on reload.

**Comment by alinearielo** [comment]
Is this still working after Figma released pay dev mode? Thanks!

**Reply by pocka** [reply]
That library uses [Figma REST API](https://www.figma.com/developers/api), which is (at least for now) not tied to Dev Mode.
So, the answer is "Yes", following "... as long as you have an access to the REST API". The screenshot is from [their pricing page](https://www.figma.com/pricing/) as of 2024-04-06.

<img width="1238" alt="Screenshot 2024-04-06 at 2 05 57" src="https://github.com/storybookjs/addon-designs/assets/13316015/30188157-ff5e-484b-bbad-819b1feb061e">

Although I have not tested since the Dev Mode drama, I believe there is no negative change affecting the library.
